### ISSUE TITLE
Add load / pipeline run metadata to RDF / blazegraph

### ISSUE THREAD

**Issue by kltm** [geneontology/pipeline#20]
Currently, while we are have a system in place for producing a blazegraph journal, we are not yet populating the journal with modeled metadata. We'll want this for obvious reasons and to help with various external metrics.

**Comment by lpalbou** [comment]
Metadata (mostly stats) that I think would be useful (either as a direct information, or to keep track of the evolution of GO-CAMs, including in QC):
- generation date
- number of production / development / review models
- number of terms (annotated)
  - terms / species
- number of genes
  - genes / species
- number of species
  - terms / species
  - genes / species
- number of relationships (enabled_by, has_input, *regulates)
- number of statements (triples coming from the models)
  - statements / species
- number of curators
- number of groups

**Comment by kltm** [comment]
@dougli1sqrd Was there a standard format we were looking at at some point?
Something zero something?
@lpalbou Would you be able to take this ticket on?

**Comment by kltm** [comment]
To clarify, these are global stats.
Naturally, only public number would be available on public endpoints.

**Comment by kltm** [comment]
@cmungall I talked to @dougli1sqrd and we couldn't come up with the metadata format/standard, maybe we got it from you originally?

**Comment by lpalbou** [comment]
@kltm yes these would be global stats for GO-CAMs, and yes we could create two different results/files, one for production and one for development models.

With the Data Commons, I'll be out this week but I could certainly look into it next week, and  I probably already have most of these queries anyway.
